build.sbt

```diff
@@ -8,8 +8,16 @@ lazy val root = (project in file("."))
     Defaults.itSettings,
     libraryDependencies ++= Seq(
       "org.scalatest" %% "scalatest" % "3.0.1" % "it,test",
-      "org.apache.spark" %% "spark-core" % "2.0.1" exclude("org.scalatest", "scalatest_2.11"),
-      "org.apache.spark" %% "spark-sql" % "2.0.1" exclude("org.scalatest", "scalatest_2.11"),
+
+      /**
+        * @sw I updated spark core and spark sql from 2.0.1 to 2.1.0 to
```

Comment:
EMR is now on 2.2.0. I think we should keep up with it, unless there's a problem we encounter.

Reply:
But we can try that outside of this PR. Just saying we shouldn't be worried about having strayed from 2.0.1.

Reply:
Thanks. To clarify, are you saying the move from 2.0.1 to 2.1.0 should be done as a separate ticket/PR?

Reply:
Nope, I think we should leave what you have, and do 2.2.0 in another ticket.